### DISCUSSION TITLE
Background and motivation for deep learning tutorials on cell type calling

### DISCUSSION THREAD

**Discussion by MiTPenguin** [scverse/spatialdata#721] [General]
Hi,

I'm following the deep learning tutorial on cell type calling using HnE images. I wonder if there are references or other background materials for this kind of approach for classifying cell types.

The tutorial itself, understandably with the small dataset, didn't show very good results. I'm interested whether the relatively simple approach (just densely connected network) can be a quick and dirty way of doing decent cell type classification with HnE images.

**Comment by LucaMarconato** [comment]
Hi, we wanted to use a very simple network to keep the focus on the dataloader; a systematic search for a highly performant network was out-of-scope. This is emphasizes by the deliberate choice to use a small dataset size in our example. Therefore I would suggest a literature review on inferring cell-types from morphology, or a systematic exploration of the alternatives.

Anyway, in our case the first approach we considered was to use a ResNet. For instance in `squidpy` (Fig 4), already a pre-trained ResNet (no finetuning) enabled finding clusters compatible with the one obtained from neighbors based methods (Louvain).

But then, because of the dataset size, we decided to train a DenseNet (for which good performance on small datasets has been observed https://arxiv.org/pdf/1608.06993).

CC @kevinyamauchi in case you have more comments to share on this.
